service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -209,10 +209,15 @@ public ResponseEntity<ApiCreateControlledAzureRelayNamespaceResult> createAzureR
   @Override
   public ResponseEntity<ApiCreatedAzureStorageContainerSasToken>
       createAzureStorageContainerSasToken(
-          UUID workspaceUuid, UUID storageContainerUuid, String sasIPRange) {
+          UUID workspaceUuid,
+          UUID storageContainerUuid,
+          String sasIpRange,
```

Comment:
Renamed to be consistent with Swagger documentation (when I wrote the unit test I first tried specifying the URL parameter as `sasIPRange` and was confused why it didn't work).